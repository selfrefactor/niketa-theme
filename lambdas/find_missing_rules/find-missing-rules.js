import { readJson } from 'fs-extra'

import { readJsonAnt } from '../../src/ants/readJson'

export async function findMissingRules(){
  const a = await readJson(`${ __dirname }/assets/lukin.json`)
  const aa = await readJsonAnt('themes/CommunicationBreakdown.json')
  console.log(aa)
  // const a = await readJsonAnt('lambdas/find_missing_rules/')
}
