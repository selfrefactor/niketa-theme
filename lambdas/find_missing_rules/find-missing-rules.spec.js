import { findMissingRules } from './find-missing-rules.js'

test('happy', async () => {
  await findMissingRules()
})
