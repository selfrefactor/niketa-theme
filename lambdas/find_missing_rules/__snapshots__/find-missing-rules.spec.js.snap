// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`happy 1`] = `
Object {
  "missingColors": Object {},
  "missingScopes": Array [],
}
`;
